mediaqueries-5/Overview.bs

```diff
@@ -2673,10 +2675,15 @@ Detecting the desire for increased or decreased color contrast from elements on
 				or some other arrangement that is neither particularly low or high contrast.
 			</div>
 
-			Note: When the User Agent can determine whether the particular forced color scheme
-			chosen by the user is high or low contrast,
-			one of ''prefers-contrast: more'' or ''prefers-contrast: less'' should match,
+			Additionally, when the user agent can determine
+			whether the forced color palette chosen by the user
+			has a high or low contrast,
+			one of ''prefers-contrast: more'' or ''prefers-contrast: less'' should match
 			in addition to ''prefers-contrast: forced''.
+			Similarly,
+			if the forced color palette chosen by the user
+			fits within one of the color schemes described by 'prefers-color-scheme',
+			the corresponding value should also match.
```

Comment:
This requirement is already specified in color-adjust, with more detail. We can probably drop it (or pull it to a Note?)

Reply:
Nice catch. I hadn't noticed color-adjust had this with more detail already. Turning into note+cross-ref.